keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c

```diff
@@ -27,7 +27,7 @@ enum layers{
 // clang-format off
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     [MAC_BASE] = LAYOUT_69_ansi(
-        KC_ESC,  KC_1,     KC_2,     KC_3,    KC_4,    KC_5,    KC_6,     KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,          KC_MUTE,
+        QK_GESC,  KC_1,     KC_2,     KC_3,    KC_4,    KC_5,    KC_6,     KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,          KC_MUTE,
```

Comment:
Why the change to `QK_GESC`?

Reply:
Sorry, my mistake, this should be my local modification ... I will remove this commit from the PR.

Reply:
done, it has been removed now.